### PR TITLE
[issue-06] create Section Title

### PR DESCRIPTION
**Pull request by rileyhgrant**
SectionTitle component. Currently using a placeholder image with a param for its path, and a static SVG. I'm just now realizing based on the reference images I should make that SVG variable too.

### CODE REVIEW

**Comment by rileyhgrant** [comment]
Oops, not quite ready for review actually. Thanks for bearing with me, I'll get better at this.

**Comment by rileyhgrant** [comment]
Turns out I have no clue how to officially deploy to test. It merged? 

Anyways, seems like merging to the test branch might not be as relevant while we're making front end components?

**Comment by alecherryy** [comment]
> Turns out I have no clue how to officially deploy to test. It merged?
> 
> Anyways, seems like merging to the test branch might not be as relevant while we're making front end components?

Not for now, but once we start creating the actual site, it'll be nice to check if links work, components render correctly, browser passes, etc.

File: src/stories/components/SectionTitle/SectionTitle.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ * )
+ */
+export const SectionTitle = ({
+  modifierClasses, url, imagePath, title, paragraph, svgClass,
+}) => {
+  const classes = ['section-title', `${modifierClasses}`].join(' ').trim();
+  const svgClasses = ['section-title__icon', `${svgClass}`].join(' ').trim();
+
+  return (
+    <Fragment>
```
Comment (alecherryy) [inline src/stories/components/SectionTitle/SectionTitle.js:30]: @rileyhgrant don't need `Fragment` if you have a wrapping `div`.

**Review by alecherryy — CHANGES_REQUESTED** [review]

File: src/stories/components/SectionTitle/SectionTitle.js (hunk 1 above)
Comment (rileyhgrant) [reply src/stories/components/SectionTitle/SectionTitle.js]: Ah, right, that makes sense. Thanks!

**Review by rileyhgrant — COMMENTED** [review]
Definitely makes sense with the not needing the fragment!

Also - were the little flag/warnings on no newline at the end of the each file automatically from Github? Just because the Linter we're using seems to want me to add newlines at the end of each file.

**Pull request merged**